addons/fastroping/CfgVehicles.hpp

```diff
@@ -221,7 +221,7 @@ class CfgVehicles {
         class UserActions {
             class Ramp_Open;
             class Ramp_Close: Ramp_Open {
-                condition = QUOTE([ARR_5(this,'CargoRamp_Open',[[0],[1],[2]])] call FUNC(canCloseRamp));
+                condition = QUOTE([ARR_3(this,'CargoRamp_Open',[[0],[1],[2]])] call FUNC(canCloseRamp));
```

Comment:
This is wrong though. There are 4 commas, so 5 arguments. Macros don't operate in SQF syntax, it's a preprocessor of raw text.